apps/charterafrica/src/components/RichText/RichText.js

```diff
@@ -99,9 +98,7 @@ const RichText = React.forwardRef(function RichText(props, ref) {
   if (!elements?.length) {
     return null;
   }
-  return (
-    <React.Fragment ref={ref}>{serialize(elements, other)}</React.Fragment>
-  );
+  return <Fragment ref={ref}>{serialize(elements, other)}</Fragment>;
```

Comment:
`Box` that you have on `FeaturedPost` should replace Fragment here. We'll then use
```jsx
    sx={{
      "& h1": ...
    }}
```

to style content as needed.